packages/blockchain/bch/src/lib/__tests__/bch.tx.spec.ts

```diff
@@ -17,30 +18,41 @@ describe('BCH transactions', () => {
   const TX_HASH = '6376496662dc94c55b0a482cbc893ca7ba640551bdf7b00179634494be5e89f3'
   const PRIVATE_KEY = 'cVX7YtgL5muLTPncHFhP95oitV1mqUUA5VeSn8HeCRJbPqipzobf'
 
-  const transactions = bchTransactions()
-
   afterEach(() => {
     jest.clearAllMocks()
   })
 
   describe('From UTXO', () => {
-    oldBtcBasedTxTestFactory.fromUTXO({
-      transactions,
-      data: {
-        validAmount: VALID_AMOUNT,
-        utxoAmount: UTXO_AMOUNT,
-        validTxData: VALID_TX_DATA,
-      },
-      skipTest: {
-        txNotFound: true,
-      },
-      mock: {
-        requestGetRawTx: mockRequestGetRawTx,
-        requestGetUtxo: () => {}, //TODO
-        requestGetUtxoNotFound: () => {}, //TODO
-        broadcast: mockedApi.blockchain.bcash.bchBroadcast,
-      },
-      getRequestBodyFromUTXO,
+    it('test', async () => {
+      const sdk = TatumBchSDK({
+        apiKey: '4cc0a7d8-265b-4fa4-8dcb-7b088bfeb627',
+      })
+      const res = await sdk.transaction.prepareSignedTransaction(
+        {
+          fromUTXO: [
+            {
+              txHash: 'c2f9e44d4cfdadb9a5269c242cff914ef57b14b053295f0a1ef76e85339282be',
+              index: 1,
+              signatureId: 'c8c7f7a4-e23b-4566-9508-f369b98ce772',
+              signatureIdIndex: 0,
+            },
+          ],
+          to: [
+            {
+              address: 'bchtest:qr5rrwc8nw59awgpxaemwq37arzg9f303u9fp2ws65',
+              value: 0.01,
+            },
+            {
+              address: 'bchtest:qzk6zxdyjgma9y2uq5untflqpa6wfpn99gxh5sdrtl',
+              value: 0.035,
+            },
+          ],
+        },
+        {
+          testnet: true,
+        },
+      )
+      console.log('res', res)
```

Comment:
you should expect some structure here - it will be the same all the time

Reply:
looks like it was as testing example for some debugging. please rollback

Reply:
changed, but need fix afterwards.

Reply:
Changed as xlm but needs to fix bug regarded to mock. For now i'm skipping. We can change it afterwards when improving testing. Ok by you?